src/core/mod.rs

```diff
@@ -307,6 +307,24 @@ impl BPF {
         }
     }
 
+    pub fn get_syscall_prefix(&mut self) -> Result<String, Error> {
+        for prefix in SYSCALL_PREFIXES.iter() {
+            if self.ksymname(prefix).is_ok() {
+                return Ok(prefix.to_owned().to_string());
```

Comment:
I think you can just do `to_string()` here?

Reply:
You are correct!